context/app/static/js/components/workspaces/CreateWorkspaceDialog/CreateWorkspaceDialog.jsx

```diff
@@ -1,28 +1,39 @@
 import React from 'react';
 import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
 
 import CreateWorkspaceInput from 'js/components/workspaces/CreateWorkspaceInput';
 import DialogModal from 'js/shared-styles/DialogModal';
+import { useStore } from 'js/shared-styles/tables/SelectableTableProvider/store';
 import { useCreateWorkspace } from './hooks';
 
 function CreateWorkspaceDialog({ handleCreateWorkspace, buttonComponent: ButtonComponent, defaultName, ...rest }) {
   const { dialogIsOpen, setDialogIsOpen, handleSubmit, handleClose, control, errors, onSubmit } = useCreateWorkspace({
     handleCreateWorkspace,
     defaultName,
   });
+  const { selectedRows } = useStore();
+  const errorMessage = selectedRows.size > 10 ? 'you have selected ...' : null;
```

Comment:
I think you removed the full error message when adopting Nick's suggestion.

Reply:
That was a bit careless, thank you for catching that.